### PR TITLE
Change language at runtime + error handling on missing internet connection

### PR DESCRIPTION
**Pull request by alminde7**

### CODE REVIEW

**Comment by alminde7** [comment]
Please ignore the low resolution flags..

File: TaskSharper.Calender.WPF/Properties/Resources.da-DK.resx [hunk 1]
```diff
@@ -117,7 +117,82 @@
...
+    <value>September</value>
+  </data>
+  <data name="Sunday" xml:space="preserve">
+    <value>Søndag</value>
+  </data>
+  <data name="This_month" xml:space="preserve">
+    <value>Denne måned</value>
+  </data>
+  <data name="This_week" xml:space="preserve">
+    <value>Denne måned</value>
```
Comment (madsmarcussen) [inline TaskSharper.Calender.WPF/Properties/Resources.da-DK.resx]: Skal rettes til "Denne uge"

**Review by madsmarcussen — CHANGES_REQUESTED** [review]
Looks good. One translation mistake.

File: TaskSharper.Calender.WPF/Properties/Resources.da-DK.resx (hunk 1 above)
Comment (alminde7) [reply TaskSharper.Calender.WPF/Properties/Resources.da-DK.resx]: Cool, det er gjort nu 👍

**Review by jeppekikt — APPROVED** [review]
You have been approved by the mighty jeppe

**Pull request merged**